### ISSUE TITLE
Workaround solution:  Decompress kernel from zstd

### ISSUE THREAD

**Issue by AndyLavr** [speed47/spectre-meltdown-checker#370]
Hi All,

```bash
--- spectre-meltdown-checker.sh.orig    2020-06-16 07:00:08.490870437 +0300
+++ spectre-meltdown-checker.sh 2020-07-11 16:41:55.628484618 +0300
@@ -1376,6 +1376,7 @@
                        try_decompress '\135\0\0\0'       xxx   unlzma  ''      xz-utils    "$1" "$mode" "$pass" && return 0
                        try_decompress '\211\114\132'     xy    'lzop'  '-d'    lzop        "$1" "$mode" "$pass" && return 0
                        try_decompress '\177ELF'          xxy   'cat'   ''      cat         "$1" "$mode" "$pass" && return 0
+                       try_decompress '(\265/\375'       xxy   unzstd  ''      zstd        "$1" "$mode" "$pass" && return 0
                done
        done
        kernel_err="kernel compression format is unknown or image is invalid"
```

Best regards, Andy Lavr.

**Comment by olifre** [comment]
Thanks, that helped me, the checker could not decompress my kernel image before. 
Maybe this issue would gain more visibility as a pull request?

**Comment by zen2** [comment]
Same problem here and this patch works.
zstd compression support need to be added.

**Comment by speed47** [comment]
Thanks for the patch, just included pushed it to main branch. Please reopen if it doesn't work as intended!
